recipes/r-spscomps/meta.yaml

```diff
@@ -0,0 +1,102 @@
+{% set version = '0.3.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-spscomps
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/spsComps_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/spsComps/spsComps_{{ version }}.tar.gz
+  sha256: 1d7e359ef62ffb543d0d0221130be1788f9654073599bf526fb52ffd7a348e0b
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  skip: true  # [r-base < 4]
```

Comment:
Does `r-base` work here? Usually we use `r_base`.

Reply:
I had tried that and it doesn't (I was surprised)

Reply:
I take that back, apparently nothing was built